src/_locales/en/messages.json

```diff
@@ -119,7 +119,7 @@
       "description": "This is alternative text for the back arrow button."
     },
     "btn-arrow-forward": {
-      "message": "Go to How Faceook Container Works",
+      "message": "Go to How Facebook Container Works",
```

Comment:
This is the change I neeed to look out for (string is translated in zh-TW)
https://pontoon.mozilla.org/zh-TW/facebook-container/all-resources/?search=btn-arrow-forward&string=198587